blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,258 @@
+
+<div class="text-center benchmark-results">
+  <img src="https://storage.googleapis.com/sourcegraph-assets/tidb-bundle-size.png" width="48%">
+  <img src="https://storage.googleapis.com/sourcegraph-assets/etcd-bundle-size.png" width="48%">
+  <br />
+  <img src="https://storage.googleapis.com/sourcegraph-assets/titan-bundle-size.png" width="48%">
+  <img src="https://storage.googleapis.com/sourcegraph-assets/zap-bundle-size.png" width="48%">
+</div>
+
+<style>
+  .blog-post__body .benchmark-results img { box-shadow: none; display: inline; margin: 10px auto; }
+</style>
+
+We plan to continue on this path of performance improvements, and the next release will focus on processing multiple bundles concurrently in order to multiply the benefit of this release’s raw latency gains.
+
+For detailed technical information about recent performance boosts, [read the blog post](https://about.sourcegraph.com/blog/performance-improvements-in-precise-code-intel) on performance improvements for precise code intelligence.
+
+
+### Automatic precise code intelligence
+
+<div class="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
+    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/BHYka1CT700?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0" allowfullscreen="" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" frameborder="0"></iframe>
+</div>
+
+An experimental feature on [sourcegraph.com/search]([https://sourcegraph.com/search](https://sourcegraph.com/search)) will automatically create an [LSIF index]([https://docs.sourcegraph.com/user/code_intelligence/lsif](https://docs.sourcegraph.com/user/code_intelligence/lsif)) for eligible repositories based on their popularity, so that repository will have precise results on hover, definition, and reference operations. We are currently able to index Go repositories containing a go.mod file that does not require additional build steps and are working on expanding the set of eligible repositories to support additional languages and more sophisticated repository structures.
+
+To see this experiment in action, pick a Go repository hosted on GitHub that has yet to be discovered. Possibly something useful with a passionate cult-following. Then, visit that repository in sourcegraph.com/search. After navigating through the code base after a while, you should see the hover tooltips and definition results become more accurate.
+
+Currently, a repository will need 50 navigation events (hovers, jump to definition, find references) to trigger the auto-indexing procedure. We are continually tuning these heuristics to bring precise results to more repositories faster.
+
+
+## Product preview: code insights
+
+TODO: Felix will make a video
```

Comment:
@felixfbecker reminder to make the video :) you can send to me and i can upload to youtube to embed here

Reply:
@felixfbecker removing this placeholder text but will insert the video once it's ready

Reply:
Added :)